src/components/Typography/Title/Title.test.tsx

```diff
@@ -0,0 +1,18 @@
+import { render } from "@testing-library/react";
+import { Title } from "./Title";
+import { ThemeProvider } from "@/theme";
+
+describe("Title", () => {
+  test("given a text, should render it", () => {
+    const text = "text to render";
+    const rendered = render(
+      <ThemeProvider theme="light">
```

Comment:
this is a pattern that probably will get repeated on all tests. We should create our test utils and create our own render method that already does this. This will dry all tests on all components.

Reply:
Agree, created an issue https://github.com/ClickHouse/click-ui/issues/64